src/Request.php

```diff
@@ -26,6 +26,8 @@
 namespace Alma\API;
 
 // In older versions of PHP (<= 5.5.19), those constants aren't defined – we do need them though
+use mysql_xdevapi\Exception;
```

Comment:
??

Reply:
it's a mistake, I pushed dev test accidentally